Java/dp/FloydWarshallAlgorithm.java

```diff
@@ -0,0 +1,129 @@
+/*
+    Floyd-Warshall Algorithm is an algorithm which is used for finding the shortest path
+    between all the pairs of vertices in a weighted graph. This algorithm works for both
+    the directed and undirected weighted graphs. But, it does not work for the graphs
+    with negative cycles (where the sum of the edges in a cycle is negative).
+    This algorithm follows the dynamic programming approach to find the shortest paths.
+    Below is the Java implementation:
+*/
+import java.util.*;
+import java.lang.*;
+import java.io.*;
+
+class FloydWarshallAlgorithm
+{
+    final static int INF = 99999, V = 4;
+    void fWarshallFunction(int graph[][])
+    {
+        int dist[][] = new int[V][V];
+        int i, j, k;
+        /*
+           Initialize the solution matrix
+           same as input graph matrix.
+           Or we can say the initial values
+           of shortest distances
+           are based on shortest paths
+           considering no intermediate
+           vertex.
+        */
+        for (i = 0; i < V; i++)
+            for (j = 0; j < V; j++)
+                dist[i][j] = graph[i][j];
+        /*
+           Add all vertices one by one
+           to the set of intermediate
+           vertices.
+          ---> Before start of an iteration,
```

Comment:
Use `-` instead of `------>`

Reply:
I thought it will help the user more, but no problem, doing the changes